test/test_special_elixirs.jl

```diff
@@ -22,6 +22,9 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
     mean_eoc = convergence_test(@__MODULE__, joinpath(EXAMPLES_DIR, "2d", "elixir_advection_extended_curved.jl"), 3)
     @test isapprox(mean_eoc[:l2], [4.0], rtol=0.01)
 
+    mean_eoc = convergence_test(@__MODULE__, joinpath(EXAMPLES_DIR, "2d", "elixir_euler_nonperiodic_p4est.jl"), 3)
```

Comment:
Did you also test `examples/2d/elixir_euler_free_stream_p4est.jl`?

Reply:
I don't think it makes much sense to test EOC for FSP, since the errors should be machine zero anyways. Or am I missing something?

Reply:
Ah, I forgot to change `free_stream_curved` to `free_stream_p4est`.